admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/JwtConfig.kt

```diff
@@ -53,7 +53,7 @@ class JwtConfig(private val config: ApplicationConfig) {
      * A lifetime of a JWT. Optional, 30 minutes by default.
```

Comment:
Comment became outdated
Just leave it w/o mentioning exact value (might be misleading, easily becomes obsolete)

Reply:
leave value for javadoc (if we add it in the future) just update to the right number